### ISSUE TITLE
LTR例子中直接运行python ranknet.py出现错误

### ISSUE THREAD

**Issue by lutaojian** [PaddlePaddle/models#159]
$ python ranknet.py 
I0713 17:47:23.595508 19546 Util.cpp:166] commandline:  --use_gpu=False --trainer_count=4 
I0713 17:47:23.610213 19546 GradientMachine.cpp:85] Initing parameters..
I0713 17:47:23.610375 19546 GradientMachine.cpp:92] Init parameters done.
Traceback (most recent call last):
  File "ranknet.py", line 134, in <module>
    train_ranknet(pass_num)
  File "ranknet.py", line 94, in train_ranknet
    num_passes=num_passes)
  File "/home/users/lutaojian/work/paddle/paddle-binary/python27/lib/python2.7/site-packages/paddle/v2/trainer.py", line 153, in train
    in_args = feeder(data_batch)
  File "/home/users/lutaojian/work/paddle/paddle-binary/python27/lib/python2.7/site-packages/py_paddle/dataprovider_converter.py", line 278, in __call__
    return self.convert(dat, argument)
  File "/home/users/lutaojian/work/paddle/paddle-binary/python27/lib/python2.7/site-packages/paddle/v2/data_feeder.py", line 134, in convert
    return DataProviderConverter.convert(self, reorder_data(dat), argument)
  File "/home/users/lutaojian/work/paddle/paddle-binary/python27/lib/python2.7/site-packages/py_paddle/dataprovider_converter.py", line 263, in convert
    scanner.pre_scan(each_step)
  File "/home/users/lutaojian/work/paddle/paddle-binary/python27/lib/python2.7/site-packages/py_paddle/dataprovider_converter.py", line 112, in pre_scan
    self.__dim__ = reduce(lambda x, y: x * y, self.__shape__)

**Comment by dzhwinter** [comment]
```
Test with Pass 1, {'__auc_evaluator_0__': 0.0}
Begin to Infer...
Traceback (most recent call last):
  File "ranknet.py", line 136, in <module>
    ranknet_infer(pass_id=pass_num-1)
  File "ranknet.py", line 126, in ranknet_infer
    output_layer=output, parameters=parameters, input=infer_data)
  File "/usr/local/lib/python2.7/dist-packages/paddle/v2/inference.py", line 132, in infer
    inferer = Inference(output_layer=output_layer, parameters=parameters)
  File "/usr/local/lib/python2.7/dist-packages/paddle/v2/inference.py", line 38, in __init__
    val.copyFromNumpyArray(parameters.get(name).flatten())
  File "/usr/local/lib/python2.7/dist-packages/paddle/v2/parameters.py", line 206, in get
    return self.__getitem__(key=parameter_name)
  File "/usr/local/lib/python2.7/dist-packages/paddle/v2/parameters.py", line 127, in __getitem__
    shape = self.get_shape(key)
  File "/usr/local/lib/python2.7/dist-packages/paddle/v2/parameters.py", line 161, in get_shape
    raise ValueError("No such parameter %s" % key)
ValueError: No such parameter ___fc_layer_4__.wbias
````

**Comment by dzhwinter** [comment]
this issue has been solved in #173
